spring-pulsar/src/main/java/org/springframework/pulsar/aot/PulsarRuntimeHints.java

```diff
@@ -29,10 +30,6 @@
 import org.apache.pulsar.client.impl.conf.ProducerConfigurationData;
 import org.apache.pulsar.client.util.SecretsSerializer;
 import org.apache.pulsar.common.protocol.Commands;
-import org.apache.pulsar.shade.io.netty.buffer.AbstractByteBufAllocator;
```

Comment:
I went ahead and moved these shaded ones in w/ their other shaded family members using the string API rather than direct imports.